src/qt/sendcoinsdialog.cpp

```diff
@@ -369,6 +369,8 @@ void SendCoinsDialog::on_sendButton_clicked()
         accept();
         CoinControlDialog::coinControl()->UnSelectAll();
         coinControlUpdateLabels();
+        // Navigate to transaction history page:
```

Comment:
Remove comment? It's only relevant below when connecting.

Reply:
Removed